nrf_security/src/CMakeLists.txt

```diff
@@ -221,6 +196,31 @@ endif()
 
 # Library for TLS protocol support
 if (CONFIG_MBEDTLS_TLS_LIBRARY)
+  # TLS files
```

Comment:
Nothing wrong with this line, I just want note that it's better to merge this commit with the previous one: 3e27e70898dcd92b3aef4f520640ecce5922e145

They have identical names and they do the same thing. No need to be separate I think.

Reply:
done